CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.12.3 (TODO)
+
+- Fix advice inputs in transaction inputs not being propagated through ([#2094](https://github.com/0xMiden/miden-base/pull/2094)).
```

Comment:
nit (feel free to disregard): This is a breaking change

Reply:
You mean because the serialization format changed or another reason? If we want to avoid that breaking change, we could also keep the field for serialization purposes but basically ignore it.